internal/runners/push/push.go

```diff
@@ -209,12 +209,24 @@ func (r *Push) Run(params PushParams) error {
 	}
 
 	// Update the project at the given commit id.
-	err = model.UpdateProjectBranchCommitWithModel(targetPjm, branch.Label, commitID)
-	if err != nil {
-		if errs.Matches(err, &model.ErrUpdateBranchAuth{}) {
-			return locale.WrapInputError(err, "push_project_branch_no_permission", "You do not have permission to push to {{.V0}}.", targetNamespace.String())
-		} else {
-			return locale.WrapError(err, "push_project_branch_commit_err", "Failed to update new project {{.V0}} to current commitID.", targetNamespace.String())
+	// Currently, the build planner doesn't support attaching a staged commit
```

Comment:
This is a blocker for this story. Please file something with extensibility to address. There's no value in us adopting this new endpoint if it's not eliminating the old one.